### PR TITLE
Apply creator to nft; multi creators/copyrighters available

### PR DESCRIPTION
**Pull request by wyuinche**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature : apply nft creator to nft

* **What is the current behavior?** (You can also link to an open issue here)
1. No nft creator in nft information
2. Only one copyrighter is available.

* **What is the new behavior (if this is a feature change)?**
1. It is available to set up multi-creator for one nft.
2. It is available to set up multi-copyrighter for one nft.

* **Other information**:
The number of nfts per one operation-item hasn't changed yet. It will be fixed in the next feature.

### CODE REVIEW

File: nft/righter.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+	"github.com/spikeekips/mitum/util/isvalid"
+)
+
+var (
+	RighterType   = hint.Type("mitum-nft-righter")
+	RighterHint   = hint.NewHint(RighterType, "v0.0.1")
+	RighterHinter = Righter{BaseHinter: hint.NewBaseHinter(RighterHint)}
+)
+
+type Righter struct {
```
Comment (soonkuk) [inline nft/righter.go:16]: @wyuinche What does the name `Righter` mean?

Comment (soonkuk) [reply nft/righter.go:16]: @wyuinche There are cases where er is added when naming, but as far as I know, it is a case of creating an interface name by attaching it to the name of a method such as Print(), Printer Read(), Reader, etc. Of course, in the current mitum model, there are cases where a type is named by appending er even if it is not an interface, but the name Righter has an inaccurate meaning as well as does not seem to follow the convention.
If you can't think of a good name right now, set it to RightHolder first and change it later.

**Comment by soonkuk** [comment]
@wyuinche 
If the modification work is finished, please post a message that the work is completed in conversation.

File: nft/righter.go (hunk 1 above)
Comment (wyuinche) [reply nft/righter.go]: @soonkuk `Righter` has been renamed `RightHolder`.

**Comment by soonkuk** [comment]
@wyuinche
There are still unresolved conversations.

File: nft/righter.go [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	hint.BaseHinter
+	account base.Address
+	signed  bool
+	clue    string
```
[the remark this answers is not in the document]
Comment (wyuinche) [reply nft/righter.go:20]: @yoonilc said that some information, such as an ethereum address, may be needed to prove the creator/copyrighter himself. So the 'clue' attribute was pre-added to serve this role in the future.

[the remark this answers is not in the document]
Comment (soonkuk) [reply nft/righter.go:20]: @wyuinche 
I think it's some kind of metadata. I think it's better to put them all at once in nft's metadata.
I don't think it's a good idea to put metadata in multiple places.

[the remark this answers is not in the document]
Comment (wyuinche) [reply nft/righter.go:20]: @soonkuk
Ok! I'll change it.

[the remark this answers is not in the document]
Comment (wyuinche) [reply nft/righter.go]: @soonkuk `clue` has been removed from `RightHolder`.

**Review by soonkuk — APPROVED** [review]

File: nft/righter.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (soonkuk) [reply nft/righter.go]: @wyuinche Ok I got it

**Pull request merged**